R/adae.R

```diff
@@ -0,0 +1,473 @@
+# Name: ADAE
+#
+# Label: Adverse Event Analysis Dataset
+#
+# Input: ae, adsl
+#
+# Developed by : Phani Tata & Joel Laxamana
```

Comment:
Please delete this file. It's very confusing to have 2 ADAE scripts in different folders.

Reply:
In the terminal.

```
git rm R/adae.R
git commit -m "Remove redundant script"
git push
```

Reply:
Done.